### PR TITLE
Feature 11571 environment import

### LINKED ISSUE #11571: Add an import mechanism to the environment directory [3]
### Feature Description

Allow users to import environment entities through the environment directory. This issue is part of the EPIC https://github.com/hzi-braunschweig/SORMAS-Project/issues/11533 on the environmental surveillance module.

### Added Value/Benefit
Users can import environments from other sources or can batch-import a lot of environments without having to go through the creation form for each one of them.

### Acceptance Criteria

- [x] An "Import" button has been added to the environment directory to the left of the "New environment" button; only visible for users with the `ENVIRONMENT_IMPORT` user right
- [x] Clicking the button brings up the usual import window
- [x] An import guide adjusted to the environment import can be downloaded (use the .docx template to create one or ask @MateStrysewske for support)
- [x] An import template with all relevant environment fields can be downloaded
- [x] Environments can be successfully imported, and an error template can be downloaded if something goes wrong
- [x] A unit test that tests this functionality with an example .csv has been added

### PR DESCRIPTION
**Pull request by leventegal-she**
Fixes #11571

### CODE REVIEW

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=12475

**Comment by sergiupacurariu** [comment]
currently, it is possible to import an environment having EnvironmentMedia set to anything but WATER and also add data to fields like WaterType, InfrastructureDetails, UseOfWater which should be filled only if the EnvironmentMedia is set to Water. This data cannot be seen in the UI due to the constraints but this data is stored in the database.  It should not be possible to import a situation like the one described above.

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=12475

File: sormas-api/src/main/resources/validations.properties [hunk 1]
```diff
@@ -273,3 +273,4 @@ removeRolesWithEditRightFromOwnUser = Cannot remove all user roles with <b>edit
 removeUserRightEditRightFromOwnUser = Cannot remove <b>edit user</b> right of currently logged in user
 removeUserEditRightFromOwnUser = Cannot remove <b>edit user right</b> right of currently logged in user
 externalMessageRefersToMultipleEntities = The external message refers to multiple entities (Either at least two cases or a case and a contact or an event participant)
+environmentWaterFieldsSetWithNotWaterMedia = The following fields can be set only if the media type is not water: %s
```
Comment (sergiupacurariu) [inline sormas-api/src/main/resources/validations.properties:276]: not should be removed "water"  instead of "not water"

**Review by sergiupacurariu — CHANGES_REQUESTED** [review]

**Review by sergiupacurariu — APPROVED** [review]

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=12475

**Pull request merged**